app/integrations/google_workspace/google_service.py

```diff
@@ -23,9 +23,19 @@
 from googleapiclient.errors import HttpError, Error  # type: ignore
 from google.auth.exceptions import RefreshError  # type: ignore
 
+# Define the default arguments
+DEFAULT_DELEGATED_ADMIN_EMAIL = os.environ.get("GOOGLE_DELEGATED_ADMIN_EMAIL")
+DEFAULT_GOOGLE_WORKSPACE_CUSTOMER_ID = os.environ.get("GOOGLE_WORKSPACE_CUSTOMER_ID")
+
 load_dotenv()
 
 
+def convert_to_camel_case(snake_str):
+    """Convert a snake_case string to camelCase."""
+    components = snake_str.split("_")
+    return components[0] + "".join(x.title() for x in components[1:])
+
+
```

Comment:
I might be totally blind (and probably am) but where do you use this function other than for the unit tests?

Reply:
upcoming PR 😅